esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/adding_a_recipe.rst

```diff
@@ -0,0 +1,81 @@
+Adding a recipe to the Recipe Test Workflow (|RTW|)
+===================================================
+
+.. include:: ../common.txt
+
+**Please note**: Before you follow these steps to add it, you must be
+able to successfully run it with the latest version of ESMValTool on the
+compute server you use at your site, as detailed by the ``platform`` option in
+the ``[[COMPUTE]]`` section in the site-specific ``.cylc`` file in the
+``esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/site/`` directory.
+
+#. Obtain the duration and memory usage of the recipe from the messages printed
+   to screen after running your recipe on the compute cluster you use at your
+   site; these messages will look something like::
+
+    YYYY-MM-DD HH:MM:SS:sss UTC [12345] INFO    Time for running the recipe was: 0:02:13.334742
+    YYYY-MM-DD HH:MM:SS:sss UTC [12345] INFO    Maximum memory used (estimate): 2.4 GB
+    [...]
+    YYYY-MM-DD HH:MM:SS:sss UTC [12345] INFO    Run was successful
+
+#. Add the recipe to the ``[task parameters]`` section in the
+   ``esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc``
+   file. If the recipe takes less than 10 minutes to run then it should be
+   added to the ``fast`` option. Recipes that take longer than ten minutes
+   should be added to the ``medium`` option.
```

Comment:
We should extend this to mention the syntax that @chrisbillowsMO raised at https://github.com/ESMValGroup/ESMValTool/pull/3614#discussion_r1617145289.

Reply:
I've had a go at describing the syntax in commit 5569fb08e